ci/clang_check_attributes.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env python3
```

Comment:
I think this is better located under "nodist" as it is usable outside the ci tool.